### PR TITLE
Add list manipulation and dijkstra algorithm.

### PR DESCRIPTION
**Pull request by MrChoclate**

### CODE REVIEW

**Comment by MrChoclate** [comment]
@Atilla106/core review?

File: public/index.html [hunk 1]
```diff
@@ -78,12 +78,86 @@
...
+dix_derniers = asso_liste[-10:]
+sans_les_dix_premiers = asso_liste[10:]
+asso_liste[3:7]  # Du 4ème au 7ème !
+						</code>
+					</pre>
+				</section>
+
+				<section>
+					<h5>Une belle fonction récursive</h5>
+					<span>Tu sens arrivé la programation dynamique?</span>
```
Comment (flavray) [inline public/index.html]: arriver*

File: public/index.html [hunk 2]
```diff
@@ -78,12 +78,86 @@
...
+    if n <= 1:
+        return 1
+    else:
+        return fib(n - 1) + fib(n - 2)
+						</code>
+					</pre>
+				</section>
+
+				<section>
+					<h5>Vous n'avez jamais autant aimer chercher dans un dictionaire !</h5>
```
Comment (flavray) [inline public/index.html]: aimé*

File: public/index.html (hunk 1 above)
Comment (MrChoclate) [inline public/index.html]: Thanks !

**Pull request merged**